index.html

```diff
@@ -225,18 +233,22 @@ <h4 class="item_name">Skills</h4>
       </li>
     </ul>
     <footer class="portofolio_footer">
-      <div class="contact_me_section">
-        <h1 class="contact_form_title">
+      <div class="contact_me_section"> 
+        <div class="completed_form">
+          <h1 class="contact_form_title">
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
```

Comment:
- Please reduce the font size of your contact_title so that it will match more to the Figma design as shown in the picture bellow
## Figma design

![image](https://user-images.githubusercontent.com/95297251/186786932-5cedd513-b61b-408d-9ea2-76103b3d7e12.png)

## your design 

![image](https://user-images.githubusercontent.com/95297251/186787035-3ccc8684-af17-4e2a-b48e-e722e2b93e1d.png)